infrastructure/via/src/helpers.ts

```diff
@@ -0,0 +1,16 @@
+import * as fs from 'fs/promises';
+import * as dotenv from 'dotenv';
+
+export async function updateEnvVariable(envFilePath: string, variableName: string, newValue: string) {
```

Comment:
This is also already present in `celestia.ts`. It makes sense to me to be a helper function, just reuse it then in celestia.ts

Reply:
Fixed